src/pocketmine/item/Item.php

```diff
@@ -524,6 +542,27 @@ public function getEnchantment(int $id){
 
 		return null;
 	}
+	
+	/**
+	* @param int $id
+	* @param int $level
+	*/
+	public function removeEnchantment(int $id, int $level = null){
```

Comment:
I would prefer an unused int value such as -1 here as opposed to `null` I don't like using `null` apart from for objects due to it encouraging unexpected behaviour.